sdk/nodejs/lambda/lambdaMixins.ts

```diff
@@ -254,6 +254,8 @@ export function createFunctionFromEventHandler<E, R>(
  * dependencies) into a form that can be used by AWS Lambda.  See
  * https://www.pulumi.com/docs/tutorials/aws/serializing-functions/ for additional
  * details on this process.
+ * This CallbackFunction will automatically create an IAM Policy that replicates the existing AWSLambdaFullAccess
+ * if no IAM Role is specified
```

Comment:
```suggestion
 * If no IAM Role is specified, CallbackFunction will automatically create an IAM Policy that replicates the now-deprecated AWSLambdaFullAccess managed policy. We do this to avoid breaking existing users who have come to rely on that set of permissions. In the next major version, we will remove this functionality and rely on the  `AWSLambda_FullAccess` managed policy.
```